File: 007_Reverse_Integer/007_2.py
class Solution:
    def reverse(self, x):
        if x < 0:
            res = -1 * int(str(-x)[::-1])
        else:
            res = int(str(x)[::-1])

        if abs(res) > 0x7fffffff:
            return 0
        return res

File: 007_Reverse_Integer/007_2.cpp
class Solution {
public:
    int reverse(int x) {
        string s = to_string(x);
        std::reverse(s.begin(), s.end());
        long long res;
        if (x < 0)
            res = -1 * stoll(s);
        else
            res = stoll(s);
        if (abs(res) > INT_MAX) return 0;
        else                    return (int)res;
    }
};